workstation.md

```diff
@@ -154,7 +154,160 @@ And you should see the exercism.io site running!  This is your local copy of the
+For this project, we will use Ruby version 2.2.2. Install it using 
+```bash
+rvm install ruby-2.2.2
+```
+
+Type `rvm list` again and it should display
+
+>   rvm rubies
+>
+>       ruby-2.2.2
+
+Now we have rvm and the correct version of Ruby installed.
+
+We have to set it as the default version to use now. Execute the following command:
+```bash
+rvm --default use 2.2.2
+```
+
+## Installing Node
+```bash
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+sudo apt-get install -y nodejs
+```
+
+## Installing and setting up PostgreSQL
+```bash
+sudo apt-get install postgresql postgresql-contrib
+```
+The setup script also requires another related package, so let's install that now
+```bash
+sudo apt-get install libpq-dev
+```
+You need to set yourself up as a postgres user. Do so with this command:
+```bash
+sudo -u postgres createuser --superuser $USER
+```
+Run postgres with the following command
+```bash
+sudo -u postgres psql postgres
+```
+You should see the postgres prompt come up. It looks like
+>postgres=# 
+
+In the postgres prompt, ensure your user was created. Do this by typing in `\du`:
+>postgres=# `\du`
+
+This will display the table of users. It should include a user called postgres and another called the same user as your username:
+
+> Role name |                         Attributes                         | Member of 
+>-----------+------------------------------------------------------------+-----------
+>
+> username  | Superuser, Create role, Create DB                          | {}
+>
+> postgres  | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
+
+Press `q` to exit the table.
+
+Next, we need to set a password for the new postgres account. It is easiest to use the same password that is specified in the exercism.io/config/database.yml because the setup script will ask you for several passwords in a row without mentioning which password is for which database user. Open the datbase.yml file with your favorite text editor (I'm partial to gedit): 
+```bash
+gedit ~/exercism.io/config/database.yml
+```
+and either change the password to whatever secure password you prefer, or note what the password in the file is. If you choose to edit the database.yml file, remember to create a `.git/info/exclude` file so your changes don't get committed to the database. Go back into postgres using `sudo -u postgres psql postgres` and at the postgres command prompt enter `\password username` where you change username to your username. Example:
+
+>postgres=# `\password aaron`
+
+This will prompt you to type in a password. Set the password to match the database.yml file's password.
+
+We need to leave postgres running in the background, so open a new terminal window to continue set up.
+
+When you are finished running the application, you can close out of postgresql with `\q`.
+
+
+## Run the setup script
+Ensure you are in ~/exercism.io
+```bash
+cd ~/exercism.io
+```
+Now run the setup script:
+```bash
+bin/setup
```

Comment:
I was prompted for my database password when running this script.  If this is expected behavior, it would be good to mention it here.